pyaedt/mechanical.py

```diff
@@ -228,9 +243,9 @@ def assign_uniform_convection(self, objects_list, convection_value, convection_u
         convection_value :
             Convection value.
         convection_unit : str, optional
-            Unit of the convection. The default is ``"w_per_m2kel"``.
+            Units for the convection value. The default is ``"w_per_m2kel"``.
         temperature : str, optional
-            Temperature. The default is ``"AmbientTemp"``.
+            Type of temperature. The default is ``"AmbientTemp"``.
```

Comment:
It's unclear if this can also be a value.  @maxcapodi78, can we also use a floating point value for temperature here?